Packs/GoogleCloudStorage/Integrations/GoogleCloudStorage/GoogleCloudStorage.yml

```
@@ -116,6 +116,11 @@ script:
     description: Deletes a bucket.
     name: gcs-delete-bucket
   - arguments:
+      defaultValue: None
+      description: use a delimiter if you want to limit results within a specific "folder", default is None
+      name: delimiter
+      description: specify to limit blobs within a "folder" i.e. "folder-1/" if blob is "folder-1/file.txt"
+      name: prefix
```

```suggestion
       description: use a delimiter if you want to limit results within a specific "folder", default is None
    - name: delimiter
       description: specify to limit blobs within a "folder" i.e. "folder-1/" if blob is "folder-1/file.txt"
    - name: prefix
```
please remove the default value and fix the yml structure